components/ledger/cmd/serve.go

```diff
@@ -14,6 +15,8 @@ const (
 	cacheEvictionPeriodFlag  = "cache-eviction-period"
 	cacheEvictionRetainDelay = "cache-eviction-retain-delay"
 	queryLimitReadLogsFlag   = "query-limit-read-logs"
+
+	ballastSizeInBytesFlag = "ballast-size-in-bytes"
```

Comment:
"-in-bytes" is maybe useless.
I think this should be mentioned on usage but not required in the flag.

Reply:
Done thanks !